statusHandler/termuic/termuiConsole.go

```diff
@@ -3,64 +3,33 @@ package termuic
 import (
```

Comment:
might move this file in statusHandler/views/termuic package

Reply:
Done.